assets/css/index.css

```diff
@@ -24,4 +29,15 @@
     height: 5rem;
     text-align: center;
     font-size: 40px;
-}
\ No newline at end of file
+    transition: background-color 0.2s ease-in-out;
+  }
+  
+  .box:hover {
+    background-color: rgba(var(--aliceblue), 0.5);
+  }
+  
+  .box:active {
+    background-color: var(--aliceblue);
+    border-radius: 5px;
+    margin: 0;
+  }
```

Comment:
Add extra new line at end

Reply:
Ok i'll fix it

Reply:
@malivinayak should i remove alert box or render innerhtml first and then alert statement after 1 second delay

Reply:
Yes you can take necessary steps to resolve issue